src/components/Article.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = ({ title, date, text }) => (
+  <article>
+
+    <h1>{title}</h1>
+
+    <span>{date}</span>
+
+    <p>{text}</p>
+
+  </article>
```

Comment:
Don't use useless empty lines.
Empty lines should be added only between multiline sibling elements.
```suggestion
  <article>
    <h1>{title}</h1>
    <span>{date}</span>
    <p>{text}</p>
  </article>
```